helm/chart-operator-chart/templates/rbac.yaml

```diff
@@ -105,9 +105,9 @@ rules:
   verbs:
   - create
 - apiGroups:
-  - networking.k8s.io
+  - scheduling.k8s.io
   resources:
-  - networkpolicies
+  - priorityclass
```

Comment:
There were 2 entries for net pols but none for the priority class. The chart used for app CRs is OK.